test/train-sets/ref/spin_off_aml_save.stderr

```diff
@@ -2,7 +2,7 @@ final_regressor = models/aml_spin_off.m
 using no cache
 Reading datafile = train-sets/automl_spin_off.txt
 num sources = 1
-Num weight bits = 18
+Num weight bits = 19
```

Comment:
Why did the num bits change?

Reply:
I updated this test to ensure it works for more models/higher bitsizes. Pretty unrelated I can revert